toolkit/tools/imagecustomizerschemacli/main.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/invopop/jsonschema"
+	"github.com/microsoft/azurelinux/toolkit/tools/imagecustomizerapi"
+)
+
+func main() {
+	if len(os.Args) < 1 {
```

Comment:
It might be goodness to use a proper args parsing library, in case we ever want to expand the args in the future.